packages/components/tests/dummy/app/templates/overrides/power-select.hbs

```diff
@@ -0,0 +1,284 @@
+{{page-title "PowerSelect Component"}}
+
+<h2 class="dummy-h2">PowerSelect</h2>
+
+<section>
+  <h3 class="dummy-h3" id="overview"><a href="#overview" class="dummy-link-section">§</a> Overview</h3>
+
+  <p class="dummy-paragraph">
+    <a target="_blank" rel="noopener noreferrer" href="https://ember-power-select.com/">PowerSelect</a>
+    is a popular Ember add-on aiming to overcome some limitations of the
+    <code class="dummy-code">&lt;select&gt;</code>
+    tag. HashiCorp Design System only provides style overrides for this add-on to keep it in line with other form
+    components.
+  </p>
+
+  <p class="dummy-banner dummy-banner--info dummy-paragraph"><FlightIcon @name="info" />Note: These style overrides
+    assume the PowerSelect component is set up in your project using the default theme.</p>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="how-to-use"><a href="#how-to-use" class="dummy-link-section">§</a> How to use</h3>
+  <p class="dummy-paragraph">
+    To use this component in your application follow
+    <a target="_blank" rel="noopener noreferrer" href="https://ember-power-select.com">the getting started guide on the
+      add-on website</a>
+    then add the PowerSelect overrides as shown below.
+  </p>
+
+  <h4 class="dummy-h4">Overrides import</h4>
+  <p class="dummy-paragraph">If you are already using
+    <a
+      target="_blank"
+      rel="noopener noreferrer"
+      href="https://github.com/hashicorp/design-system/blob/main/packages/components/README.md"
+    >design-system-components</a>
+    in your project the overrides import would look something like this:</p>
+  {{! prettier-ignore-start }}
+  {{! template-lint-disable no-unbalanced-curlies }}
+  <CodeBlock
+    @code='
+      @import "@hashicorp/design-system-components";
+      @import "@hashicorp/power-select-overrides";
+    '
+  />
+
+  <p class="dummy-paragraph">If you are not yet using
+    <a
+      target="_blank"
+      rel="noopener noreferrer"
+      href="https://github.com/hashicorp/design-system/blob/main/packages/components/README.md"
+    >design-system-components</a>
+    in your project but you want to use these PowerSelect custom styles you will need to add
+    <a
+      target="_blank"
+      rel="noopener noreferrer"
+      href="https://github.com/hashicorp/design-system/blob/main/packages/tokens/README.md"
+    >design-system-tokens</a>
+    to your project.</p>
+
+  <h4 class="dummy-h4">Basic usage</h4>
+
+  <p class="dummy-paragraph">As these overrides rely on specificity, to apply them you need to wrap each PowerSelect
+    instance in an element with
+    <code class="dummy-code">hds-power-select</code>
+    class applied to it.</p>
+  <p class="dummy-paragraph">Invocation of the component with overrides would look something like this:</p>
+
+  <CodeBlock
+    @language="markup"
+    @code='
+    <div class="hds-power-select">
```

Comment:
Question: Is it not possible/desirable to add a class name to the "PowerSelect" component itself vs. wrapping it with an extra div?

Reply:
Great question, @KristinLBradley! I would've loved that to be possible, but applying a `class="hds-power-select"` attribute to a `<PowerSelect>` instance results in something like this:
```
<div class="ember-basic-dropdown">
  <div class="ember-view ember-basic-dropdown-trigger hds-power-select">...</div>
  <div class="ember-basic-dropdown-content">...</div>
</div>
```
So the class is not being applied to the outermost element, but to the dropdown trigger, which doesn't help with styling the dropdown content.

Reply:
We could use adjacent sibling selector, but we needed some adjustments to the element that holds the `ember-basic-dropdown` hence I decided to use that additional wrapper element, which you can imagine I'm not fond of.

Reply:
I see. Thanks for explaining.